lib/notification.js

```diff
@@ -12,6 +12,24 @@ var NOTIFIER_NAME = "Bugsnag Node Notifier",
     NOTIFIER_URL = "https://github.com/bugsnag/bugsnag-node",
     SUPPORTED_SEVERITIES = ["error", "warning", "info"];
 
+function copyCustomExceptionProps(exceptions, metaData) {
+    for (var i = 0; i < exceptions.length; i++) {
+        if (exceptions[i].metaData) {
+            if (!metaData) {
+                metaData = {};
+            }
+            if (!Array.isArray(metaData.CustomExceptions)) {
+                metaData.CustomExceptions = [];
```

Comment:
I've made the arbitrary choice to include these in a new tab called "Custom Exceptions", not sure if there is a way to shoehorn them into the existing "Custom" tab.